doc/neps/nep-0016-benchmark.py

```diff
@@ -1,4 +1,6 @@
+# The Python (perf) module is a toolkit to write, run and analyze benchmarks
 import perf
+# abc(abstract base class, ABCMeta metaclass provides a method called register method that can be invoked by its instance)
```

Comment:
Describing what built-in packages do is the job of the Python documentation, not comments in numpy.